data/mods/Magiclysm/Spells/black_dragon.json

```
@@ -0,0 +1,68 @@
+[
+  {
+    "type": "SPELL",
+    "id": "acid_claw",
+    "name": "Acid Claw",
+    "description": "Your claws drip with acid as you make a slash attack.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SOMATIC", "VERBAL", "NO_LEGS" ],
+    "spell_class": "DRAGON_TALONS_BLACK",
+    "effect": "target_attack",
+    "damage_type": "acid",
+    "max_level": 4,
+    "min_damage": 10,
+    "max_damage": 50,
+    "damage_increment": 10.0,
+    "min_range": 1,
+    "max_range": 1,
+    "difficulty": 0,
+    "base_casting_time": 100,
+    "energy_source": "STAMINA",
+    "base_energy_cost": 100,
```

```suggestion
    "base_energy_cost": 400,
```
being able to cast this a couple dozen times a day and one-hit many zombies means this is waaay too strong. let's bump up the casting cost and i think it'll be ok

---

when I was testing the strength of it, it wasn't one-shotting any basic zombies, let along anything advanced in evolution. It may be able to one-shot a predator or something at max level, but there are no books to level it up with and simply casting spells takes a really long time to level them up with. at max level, it's dealing less damage than an ice spike, and comes at the opportunity cost of requiring melee range rather than being able to be cast from 16 spaces away.

---

ok, you've convinced me